.travis.yml

```diff
@@ -7,7 +7,7 @@ services:
 language: go
 
 go:
-  - "1.10.x"
+  - "1.11"
```

Comment:
This needs to be changed to `1.11.x` once the first patch release for Go 1.11 is released

Reply:
Can you update this now? Now that 1.2 is out I think if you change this to 1.11.x we can merge this.